presto-docs/src/main/sphinx/release/release-0.201.rst

```diff
@@ -0,0 +1,41 @@
+=============
+Release 0.201
+=============
+
+General Changes
+---------------
+
+* Improve performance of queries that process structural types and contain joins, aggregations
+  or table writes.
+* Fix planning error when column names are reused in ORDER BY query.
+* Add support for ``current_user``.
```

Comment:
Let's link to the documentation. If we can't link to the function, we can link to the chapter since it only has one function.